### PR TITLE
General rewording/grammar/font cleanup of "Revision Selection"

### PR DESCRIPTION
**Pull request by rpjday**
As usual:

  - standardize on " -- " for em dashes
  - rewording for clarity
  - add a TIP or two

### CODE REVIEW

File: book/07-git-tools/sections/revision-selection.asc [hunk 1]
```diff
@@ -38,7 +38,7 @@ Date:   Thu Dec 11 14:58:32 2008 -0800
     added some blame and merge stuff
 ----
 
-In this case, choose `1c002dd...`. If you `git show` that commit, the following commands are equivalent (assuming the shorter versions are unambiguous):
+In this case, say you're interested in the commit whose hash begins with `1c002dd...`. You can inspect that commit with any of the following variations of `git show` (assuming the shorter versions are unambiguous):
```
Comment (ben) [inline book/07-git-tools/sections/revision-selection.asc]: Could you split these sentences onto two lines please?

File: book/07-git-tools/sections/revision-selection.asc [hunk 2]
```diff
@@ -171,9 +169,15 @@ Date:   Thu Dec 11 15:08:43 2008 -0800
     Merge commit 'phedders/rdocs'
 ----
 
-It’s important to note that the reflog information is strictly local – it’s a log of what you’ve done in your repository.
-The references won’t be the same on someone else’s copy of the repository; and right after you initially clone a repository, you'll have an empty reflog, as no activity has occurred yet in your repository.
-Running `git show HEAD@{2.months.ago}` will work only if you cloned the project at least two months ago – if you cloned it five minutes ago, you’ll get no results.
+It’s important to note that reflog information is strictly local -- it’s a log only of what _you've_ done in _your_ repository.
+The references won’t be the same on someone else’s copy of the repository; also, right after you initially clone a repository, you'll have an empty reflog, as no activity has occurred yet in your repository.
+Running `git show HEAD@{2.months.ago}` will work only if you cloned the project at least two months ago -- if you cloned it five minutes ago, you’ll get no results.
```
Comment (ben) [inline book/07-git-tools/sections/revision-selection.asc]: I think what this does is find the commit _closest_ to the date/time you specify. So if I have a two-weeks-old project and I do `git show HEAD@{2.years.ago}` I'll get the initial commit.

File: book/07-git-tools/sections/revision-selection.asc (hunk 1 above)
Comment (rpjday) [reply book/07-git-tools/sections/revision-selection.asc]: Damn, screwed that up, will fix.

File: book/07-git-tools/sections/revision-selection.asc (hunk 2 above)
Comment (rpjday) [reply book/07-git-tools/sections/revision-selection.asc]: Let me verify that behaviour.

Comment (rpjday) [reply book/07-git-tools/sections/revision-selection.asc]: Yup, that's what it appears to do. You do know that that was *your* sentence, right? :-)

Comment (ben) [reply book/07-git-tools/sections/revision-selection.asc]: Yup. 😊

File: book/07-git-tools/sections/revision-selection.asc (hunk 1 above)
Comment (ben) [reply book/07-git-tools/sections/revision-selection.asc]: It was wrong to begin with. Thanks for fixing it up!

Comment (rpjday) [reply book/07-git-tools/sections/revision-selection.asc]: Ah, you're right, so it was. Heh.

**Pull request merged**